pkg/services/ngalert/tests/manager_test.go

```diff
@@ -16,6 +17,8 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+var NilMetrics = metrics.NewMetrics(nil)
```

Comment:
This shouldn't be exported should it?
```suggestion
var nilMetrics = metrics.NewMetrics(nil)
```